packages/rum/README.md

```diff
@@ -572,3 +472,4 @@ window.DD_RUM && window.DD_RUM.getInternalContext() // { session_id: "xxxx", app
 [18]: https://docs.datadoghq.com/real_user_monitoring/session_replay/privacy_options
 [19]: https://docs.datadoghq.com/getting_started/tagging/using_tags
 [20]: https://docs.datadoghq.com/real_user_monitoring/frustration_signals/
+[21]: https://docs.datadoghq.com/real_user_monitoring/guide/sampling-browser-and-browser-premium/
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏this doc must be merged before merging this PR

Reply:
My linked PR (https://github.com/DataDog/documentation/pull/15038) has been merged!